tests/python/pass/test_pass_simplify_expr.py

```diff
@@ -260,5 +266,34 @@ def expected():
     assert tvm.ir.structural_equal(mod["main"], expected()), raf.ir.AsText(mod["main"])
 
 
+def test_dropout():
+    device = "cpu"
+    shape = (10, 5)
+    dropout_op = getattr(raf._op.sym, "_contrib_dropout")
```

Comment:
maybe more straightforward?
```suggestion
    dropout_op = raf._op.sym._contrib_dropout
```